LEAF_Request_Portal/templates/view_reports.tpl

```diff
@@ -187,22 +187,51 @@ function addHeader(column) {
                          }});
             break;
         case 'days_since_last_action':
+        case 'days_since_last_step_movement':
             leafSearch.getLeafFormQuery().join('action_history');
-            headers.push({name: 'Days since last action', indicatorID: 'daysSinceLastAction', editable: false, callback: function(data, blob) {
-                var daysSinceAction;
-                if(blob[data.recordID].action_history != undefined) {
-                    var lastAction = blob[data.recordID].action_history[blob[data.recordID].action_history.length - 1];
-                    var date = new Date(lastAction.time * 1000);
-                    daysSinceAction = Math.round((today.getTime() - date.getTime()) / 86400000);
-                    if(blob[data.recordID].submitted == 0) {
+            leafSearch.getLeafFormQuery().join('stepFulfillmentOnly');
+
+            let headerName = (column == 'days_since_last_action') ? 'Days Since Last Action' : 'Days Since Last Workflow Change';
+            let indicatorIDName = (column == 'days_since_last_action') ? 'daysSinceLastAction' : 'daysSinceLastStepMovement';
+            headers.push({
+                name: headerName,
+                indicatorID: indicatorIDName,
+                editable: false,
+                callback: function(data, blob) {
+                    let daysSinceAction;
+                    let recordBlob = blob[data.recordID];
+                    if(recordBlob.action_history != undefined) {
```

Comment:
I know this is likely legacy, but please look at pulling it out into its own function.  Since we're not going to overload the QA team this time around, we have some time for cleaning up old code.  Just make sure to log the time into the "cleanup" Jira and not the one for this issue.